app/jobs/regular/pull_hotlinked_images.rb

```diff
@@ -107,6 +107,11 @@ def execute(args)
         images = doc.css("img[src]") - doc.css("img.avatar")
         images.each do |tag|
           src = tag['src']
+          if tag['class'] == 'thumbnail' && tag.parent['class'] == 'onebox-body' &&
```

Comment:
Wouldn't it be simpler to do this instead?

```
images = doc.css("img[src]") - doc.css("img.avatar") - doc.css(".onebox-body img.thumbnail")
```

Reply:
I don't want to skip those onebox images. I am finding it and removing from cooked html using Nokogiri.